File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Search/CancelPenalty.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Search

    Public Class CancelPenalty

        <JsonProperty("currency")>
        Public Property CurrencyCode As String

        <JsonProperty("start")>
        Public Property CancelStartDate As Date

        <JsonProperty("end")>
        Public Property CancelEndDate As Date

        <JsonProperty("amount")>
        Public Property Amount As Decimal

        <JsonProperty("nights")>
        Public Property Nights As Integer

        <JsonProperty("percent")>
        Public Property Percent As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Search/CancelPenalty.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses.Search
{
    using System;
    using Newtonsoft.Json;


    public class CancelPenalty
    {

        [JsonProperty("currency")]
        public string CurrencyCode { get; set; }

        [JsonProperty("start")]
        public DateTime CancelStartDate { get; set; }

        [JsonProperty("end")]
        public DateTime CancelEndDate { get; set; }

        [JsonProperty("amount")]
        public decimal Amount { get; set; }

        [JsonProperty("nights")]
        public int Nights { get; set; }

        [JsonProperty("percent")]
        public string Percent { get; set; }

    }

}